portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,23 +28,30 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> myComments = new ArrayList<>();
```

Comment:
A couple of things here (may have left this comment before):

```suggestion
  private final List<String> myComments = new ArrayList<>();
```

1. Prefer referring to [objects by their interfaces](go/ej3e-64)(if that go link doesn't resolve it's in the PDF i shared previously - item 64) unless there's a reason not to.

2. In general mutable collections can be a source of bugs. I think that [guava's immutable collections](https://github.com/google/guava/wiki/ImmutableCollectionsExplained) will come in handy (as will guava, generally). No action needed, just more food for thought.

Reply:
Thanks for the comment! Updated!